js/data/data_source/data_source.js

```diff
@@ -190,7 +190,7 @@ var DataSource = Class.inherit({
         /**
         * @name DataSourceOptions_store
         * @publicName store
-        * @type Store|Array<any>|Object
+        * @type Store|StoreOptions|Array<any>|Object|any
```

Comment:
What does `Object` value mean here? I suspect that `Object` is `StoreOptions` here.

/cc @AlekseyMartynov

Reply:
https://github.com/DevExpress/DevExtreme/blob/9cb3052fa31bca1021805b79a16e08e37a76e4bc/js/data/data_source/data_source.js#L116-L126

`store` can be:
- array
- store options
- a plain object with the `load` member

I suggest that we keep `any` and remove `Object`

Reply:
Can the store be the instance of Store object?

Reply:
yes, I see https://github.com/DevExpress/DevExtreme/blob/9cb3052fa31bca1021805b79a16e08e37a76e4bc/js/data/data_source/data_source.js#L106